app/src/main/java/com/example/compose_clean_arch/navegation/NavegationGragh.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.compose_clean_arch.navigation
+
+import androidx.compose.runtime.Composable
+
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import com.example.compose_clean_arch.data.nave.NavRoutes
+import com.example.compose_clean_arch.screens.home.MainScreen
+import com.example.compose_clean_arch.screens.welcomeApp.WelcomeScreen
+
+@Composable
+fun NavigationGraph(navController: NavHostController) {
+    NavHost(
+        navController = navController,
+        startDestination = NavRoutes.Main.route
+    ) {
+        composable(NavRoutes.Main.route) {
+            MainScreen(navController)
+        }
+            composable(NavRoutes.Welcome.route) {
+                WelcomeScreen()
+            }
```

Comment:
Indentation

Reply:
I apologize for this, but it does not appear in my code.
![image](https://github.com/batooldshilleh/ComposeCleanArch/assets/93814390/fc8f05a0-9c9c-4360-a929-24ba6ecaeae8)